src/components/pages/Users/Users.styles.js

```
@@ -0,0 +1,60 @@
+import { css } from 'styled-components'
+import styled from 'styled-components/macro'
+import theme from '../../../theme'
+import { hoverState, mediaQueryPhoneOnly } from '../../../library/styling/mediaQueries'
+import { IconAlert } from '../../icons'
+import { Td } from '../../generic/Table/table'
+import { UserIcon } from '../../UserIcon/UserIcon'
```

ESLint

[error] 'UserIcon' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)